### DISCUSSION TITLE
UTM to Latitude/Longitude conversion

### DISCUSSION THREAD

**Discussion by ganeshwarikanagaraj** [angularsen/UnitsNet#1387] [Q&A]
Does the UnitsNet library allow for the conversion of UTM to Latitude/Longitude?

**Comment by angularsen** [comment]
Hi, no this is probably too complicated to fit in the structure of UnitsNet with simple unit conversions.

I have good experience with this library though, both for EF Core + SQL and plain working with spatial data:
- https://github.com/NetTopologySuite/NetTopologySuite
- https://learn.microsoft.com/en-us/ef/core/modeling/spatial
